tests/e2e/specs/empty-trash-restore-trashed-posts.test.js

```diff
@@ -0,0 +1,71 @@
+import {
+    visitAdminPage,
+    createNewPost,
+    trashAllPosts,
+    publishPost
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Empty Trash', () => {
+
+
+    async function createPost() {
+
+        // Create a Post
+        const title = 'Test Title';
+        await createNewPost( { title } );
+        await publishPost();
+
+    }
+
+
+    afterEach( async () => {
+        await trashAllPosts();
+    } );
+
+    it( 'Empty Trash', async () => {
+
+        await createPost();
+       
+        await visitAdminPage( '/edit.php' );  
+
+        // Move post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Empty trash 
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");  
+        const deleteAllButton = await page.waitForSelector('input[value="Empty Trash"]');
+        await deleteAllButton.click();
+        await page.waitForSelector("#message");
+        
+    } );
+
+
+    it( 'Restore trash post', async () => {
+
+        await createPost();
+    
+        await visitAdminPage( '/edit.php' );  
+
+        // Move one post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Remove post from trash 
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.page-title');
+        await page.click(".untrash");
```

Comment:
I think we missed [this](https://github.com/WordPress/wordpress-develop/pull/2186#discussion_r815960434) 😅 .

Reply:
Hi @kevin940726,

This feedback is addressed